src/app/static/src/app/store/root/mutations.ts

```diff
@@ -8,5 +8,8 @@ export interface RootMutations {
 export const mutations: MutationTree<RootState> & RootMutations = {
     Reset(state: RootState) {
         Object.assign(state, emptyState);
+        state.surveyAndProgram.ready = true;
```

Comment:
Realised that if the state is reset it also reverts to not ready and gets stuck in an endless loading spinner!